editor/components/block-list/style.scss

```diff
@@ -744,6 +744,7 @@
 		opacity: 0;
 		pointer-events: none;
 
+		&:hover,
```

Comment:
I tested this with the keyboard too and it worked, just FYI.

Cool thought, I wish I thought of this!